### PR TITLE
Snails? Snails.

### PR DESCRIPTION
**Pull request by UselessTheremin**
### Intent of your Pull Request

snails

### Why is this change good for the game?

snails are pretty good

### Briefly describe your PR and the impacts of it, in layman's terms. 

roundstart snail

### What should players be aware of when it comes to the changes your PR is implementing?

snails when? snails now.

### What general grouping does this PR fall under? 
For example, Engineering changes, Medical rebalancing, TEG tweaks, etc.?

🐌 

:cl:  
tweak: snails roundstart race
/:cl:

### CODE REVIEW

**Comment by MegaEmpirical** [comment]
Does this remove the issue with their backpacks not being removable? Also, any changes to them being able to slip the floor?

Also, if we tweak the issues I'd be up for making this a donor thing. Snails have been a big part of the community for a while and as long as they have only negative stuff but are kind of amusing to apply, I think they'd be an interesting donor race.

**Comment by UselessTheremin** [comment]
ok imma fix them then make em donor only probably
